### DISCUSSION TITLE
COG Image disappears when zoomed out to world extent

### DISCUSSION THREAD

**Discussion by raster-blaster** [developmentseed/titiler#673] [General]
When viewing a COG in Titiler using the /cog/viewer, I noticed that the image is visible at a lower zoom level but disappears as I zoom out further. The same behaviour occurs in our front-end application. However, in QGIS, the COG image remains visible even when zoomed out to the world extent. I would appreciate any guidance on how to ensure the COG image stays visible when zoomed out. Thank you for your assistance.

When zoomed in, Image displays
<img width="1603" alt="Pasted Graphic 2" src="https://github.com/developmentseed/titiler/assets/11376089/d6856d5a-e513-4e71-b2e2-f0fd271831d8">

When zoomed out, Image disappears
<img width="1416" alt="Pasted Graphic 3" src="https://github.com/developmentseed/titiler/assets/11376089/5331abb2-7b83-456c-a3cf-96b31727206b">

**Comment by vincentsarago** [comment]
when using titiler, we assume a dataset has a minzoom and a maxzoom (for a given TileMatrixSet). The minzoom is calculated in rio-tiler (https://github.com/cogeotiff/rio-tiler/blob/main/rio_tiler/io/rasterio.py#L168-L188) as the resolution of the smallest overview (theoretical). Beyond this minzoom, reading a COG might be pretty inefficient (ref: https://github.com/cogeotiff/rio-tiler/issues/348) but you can `force` this in your application by ignoring the `minzoom` returned by the tilejson endpoint or by overwriting the dataset minzoom (using query parameters https://developmentseed.org/titiler/endpoints/cog/#tilesjson)
